firstvoices/backend/search/indices/dictionary_entry_document.py

```diff
@@ -268,47 +254,3 @@ def update_categories(sender, instance, **kwargs):
                 dictionary_entry.id,
             )
         )
-
-
-# If a site's visibility is changed, update all docs from index related to site
-@receiver(pre_save, sender=Site)
```

Comment:
Is there a ticket logged to deal with the necessary search index updates when a site's visibility is changed? I couldn't find one.

Reply:
Ohhh nevermind, I see where this got moved to :)